src/Main.java

```diff
@@ -1,15 +1,29 @@
+import Algorithm.Greedy.GreedyAlgorithm;
 import Loader.Loader;
 
 import java.io.IOException;
+import java.util.*;
+
 import Loader.*;
 
 
 public class Main {
-
     public static void main(String[] args) throws IOException {
         Loader loader = new Loader();
-        CVRPModel model = loader.loadFileData("Medium - X-n200-k36");
-        submitLoader(model);
+        CVRPModel model = loader.loadFileData("Medium - tai100c");
+        Random random = new Random();
+        double minValue = Double.MAX_VALUE;
+        int startPoint;
+        for (int i = 0; i < 100; i++) {
+            startPoint = random.nextInt(100);
```

Comment:
To nie wiem, czy jest tylko do testów, ale chyba bez sensu losować random, bo nie zawsze dostaniemy wynik najlepszy, tylko finalnie to raczej start point to będzie i (że za każdym razem inny wierzchołek jest pierwszy). Tutaj też trzeba uważać żeby pierwszym punktem nie było miejsce dystrybucji.